src/components/Countdown.js

```diff
@@ -36,7 +36,7 @@ const Countdown = ({ start, end, isPreview, text, updateContestStatus }) => {
           <span className="minutes">{timeLeft.mm}</span>{" "}
           <span className="seconds">{timeLeft.ss}</span>
         </span>
-        {text && timeLeft.total > 0 ? (
+        {text !== false && timeLeft.total > 0 ? (
```

Comment:
I had a misunderstanding of how this prop was being used and assumed !== false could be replaced with a truthy check.